package.json

```diff
@@ -64,7 +63,11 @@
     "flow-bin": "^0.42.0",
     "husky": "^0.13.3",
     "jest": "^19.0.2",
-    "lint-staged": "^3.4.0"
+    "lint-staged": "^3.4.0",
+    "babel-preset-react-native": "^2.0.0"
+  },
+  "peerDependencies": {
+    "babel-preset-react-native": "^2.0.0"
```

Comment:
> we don't want to bump the preset version each time it changes manually

That's fair, but do we need a peer dependency? React Native doesn't have it as a peer dep AFAIK, so it's one extra step